libyul/AsmAnalysis.cpp

```diff
@@ -309,6 +311,11 @@ std::vector<YulString> AsmAnalyzer::operator()(FunctionCall const& _funCall)
 	std::vector<YulString> const* returnTypes = nullptr;
 	std::vector<std::optional<LiteralKind>> const* literalArguments = nullptr;
 
+	auto isTransientStorageWarning = [](std::shared_ptr<Error const> error) -> bool {
+		yulAssert(error.get(), "");
+		return error->errorId() == 2394_error;
```

Comment:
I guess this could be avoided by identifying the error via the message (or at least part of it)... That way the workaround in `error_codes.py` would not be needed.

Reply:
You can also just use ``ErrorId{2394}`` here instead and already don't need the change to ``error_code.py``, can't you :-)?

Reply:
I tried that, but I guess needed a cast. I will try that again.

Reply:
Yeah, this works. Not sure what I tried before, but obviously was something different :sweat_smile: